doc/source/examples/multiphysics/dam-break-VOF/dam-break-VOF.rst

```diff
@@ -0,0 +1,286 @@
+    end
+
+In the ``initial condition``, the initial velocity and initial position 
+of the liquid phase are defined. The liquid phase is initially 
+defined as rectangle of length :math:`= 3.5` and height :math:`= 7`.
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Initial condition
+    #---------------------------------------------------
+    subsection initial conditions
+        set type = nodal
+        subsection uvwp
+            set Function expression = 0; 0; 0
+        end
+        subsection VOF
+             set Function expression = if (x<3.5 & y<7 , 1, 0)
+        end
+    end
+
+The ``source term`` subsection defines the gravitational acceleration:
+
+.. code-block:: text
+    
+    #---------------------------------------------------
+    # Source term
+    #---------------------------------------------------
+    subsection source term
+        set enable = true
+        subsection xyz
+            set Function expression = 0;-1.0; 0
+        end
+    end
+
+Two fluids are present in this simulation, hence in the ``physical 
+properties`` subsection, their physical properties should be defined:
+
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Physical Properties
+    #---------------------------------------------------
+    subsection physical properties
+        set number of fluids         = 2
+        subsection fluid 0
+            set density              = 0.02
+            set kinematic viscosity  = 0.1
+        end
+        subsection fluid 1
+            set density              = 0.9982
+            set kinematic viscosity  = 0.01
+        end
+    end
+
+We define two fluids here simply by setting the number of fluids to be :math:`2`.
+In ``subsection fluid 0``, we set the density and the kinematic viscosity for the first phase. 
+Similar procedure is done for the secondary phase in ``subsection fluid 1``. 
+
+
+In the ``mesh adaptation subsection``, adaptive mesh refinement is 
+defined for ``velocity``. ``min refinement level`` and ``max refinement 
+level`` are 3 and 5, respectively.
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Mesh Adaptation
+    #---------------------------------------------------
+    subsection mesh adaptation
+        set type                    = kelly
+        set variable                = phase
+        set fraction type           = fraction
+        set max refinement level    = 5
+        set min refinement level    = 3
+        set frequency               = 1
+        set fraction refinement     = 0.95
+        set fraction coarsening     = 0.02
+    end
+
+
+
+Call the gls_navier_stokes_2d by invoking:  
+
+``mpirun -np 2 gls_navier_stokes_2d gls_VOF_dam-break_Martin_and_Moyce.prm``
+
+to run the simulation using two CPU cores. Feel free to use more.
+
+
+.. warning:: 
+    The code will compute :math:`35,000+` dofs for :math:`600+` time 
```

Comment:
change for the correct dof values (the prm has changed)

Reply:
Those are the correct ones